sky/serve/serve_state.py

```diff
@@ -68,6 +68,9 @@ def create_table(cursor: 'sqlite3.Cursor', conn: 'sqlite3.Connection') -> None:
 db_utils.add_column_to_table(_DB.cursor, _DB.conn, 'services',
                              'active_versions',
                              f'TEXT DEFAULT {json.dumps([])!r}')
+# Whether the service's load balancer is encrypted with TLS.
+db_utils.add_column_to_table(_DB.cursor, _DB.conn, 'services', 'tls_encrypted',
+                             'INTEGER DEFAULT NULL')
```

Comment:
Should we add default value to existing entries?

Reply:
We already have a NULL default? Though i changed this to 0 to avoid confusion 🫡